po/lxd.pot

```diff
@@ -346,7 +346,11 @@ msgstr  ""
 msgid   "If this is your first run, you will need to import images using the 'lxd-images' script."
 msgstr  ""
 
-#: lxc/image.go:335
+#: lxc/image.go:215
+msgid   "Image copied sucessfully!"
```

Comment:
Same as above. :): Chiming in here. sorry for that, but this should be "successfully". Spelling mistakes look rather unprofessional. :)